2013/09/12 01:09:42 [error] [exception.CException] exception 'CException' with message 'Не определено свойство "CinemaBanner.c_published".' in Q:\home\yakutsk-back.dv\framework\base\CComponent.php:130
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\db\ar\CActiveRecord.php(145): CComponent->__get('c_published')
#1 Q:\home\yakutsk-back.dv\public_html\protected\models\CinemaBanner.php(88): CActiveRecord->__get('c_published')
#2 Q:\home\yakutsk-back.dv\public_html\protected\modules\backend\views\cinemaBanner\admin.php(45): CinemaBanner->search()
#3 Q:\home\yakutsk-back.dv\framework\web\CBaseController.php(126): require('Q:\home\yakutsk...')
#4 Q:\home\yakutsk-back.dv\framework\web\CBaseController.php(95): CBaseController->renderInternal('Q:\home\yakutsk...', Array, true)
#5 Q:\home\yakutsk-back.dv\framework\web\CController.php(869): CBaseController->renderFile('Q:\home\yakutsk...', Array, true)
#6 Q:\home\yakutsk-back.dv\framework\web\CController.php(782): CController->renderPartial('admin', Array, true)
#7 Q:\home\yakutsk-back.dv\public_html\protected\modules\backend\controllers\CinemaBannerController.php(144): CController->render('admin', Array)
#8 Q:\home\yakutsk-back.dv\framework\web\actions\CInlineAction.php(49): CinemaBannerController->actionAdmin()
#9 Q:\home\yakutsk-back.dv\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 Q:\home\yakutsk-back.dv\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#11 Q:\home\yakutsk-back.dv\framework\web\filters\CFilter.php(40): CFilterChain->run()
#12 Q:\home\yakutsk-back.dv\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#13 Q:\home\yakutsk-back.dv\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#14 Q:\home\yakutsk-back.dv\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#15 Q:\home\yakutsk-back.dv\framework\web\CController.php(291): CFilterChain->run()
#16 Q:\home\yakutsk-back.dv\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#17 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(282): CController->run('admin')
#18 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('backend/cinemab...')
#19 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#20 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#21 {main}
REQUEST_URI=/index.php/backend/cinemabanner/admin
HTTP_REFERER=http://yakutsk-back.dv/index.php/backend/cinemabanner/
---
2013/09/12 01:09:42 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 01:10:18 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 01:10:26 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 01:10:29 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 01:10:33 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 01:10:49 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 01:11:00 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 01:17:00 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 01:17:02 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 01:17:54 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 01:17:58 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 01:18:02 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 01:18:04 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 01:18:08 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 01:19:46 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 01:19:50 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 01:19:56 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 01:20:01 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 01:20:41 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 01:21:00 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 01:21:02 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 01:21:16 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 01:21:18 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 01:21:19 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 01:21:23 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 01:21:26 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 01:21:30 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 01:21:38 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 01:21:44 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 01:21:48 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 01:22:45 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 01:22:47 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 01:22:50 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 01:22:52 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 01:22:56 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 01:23:00 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 01:25:25 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 01:25:28 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 01:25:33 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 01:25:36 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 01:25:38 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 01:25:39 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 01:25:47 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 01:25:52 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 01:35:23 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 01:35:28 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 01:35:30 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 01:35:31 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 01:35:38 [error] [system.db.CDbCommand] Не удалось выполнить CDbCommand::fetchAll(): SQLSTATE[42S02]: Base table or view not found: 1146 Table 'now-yakutsk.restaurants' doesn't exist. Выполнявшийся SQL-запрос: SHOW FULL COLUMNS FROM `restaurants`.
2013/09/12 01:35:38 [error] [exception.CDbException] exception 'CDbException' with message 'Таблица "restaurants", упомянутая в записи active record класса "Restaurants", не найдена в базе данных.' in Q:\home\yakutsk-back.dv\framework\db\ar\CActiveRecord.php:2365
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\db\ar\CActiveRecord.php(411): CActiveRecordMetaData->__construct(Object(Restaurants))
#1 Q:\home\yakutsk-back.dv\framework\db\ar\CActiveRecord.php(661): CActiveRecord->getMetaData()
#2 Q:\home\yakutsk-back.dv\framework\db\ar\CActiveRecord.php(1604): CActiveRecord->getTableSchema()
#3 Q:\home\yakutsk-back.dv\framework\web\CActiveDataProvider.php(228): CActiveRecord->count(Object(CDbCriteria))
#4 Q:\home\yakutsk-back.dv\framework\web\CDataProvider.php(224): CActiveDataProvider->calculateTotalItemCount()
#5 Q:\home\yakutsk-back.dv\framework\web\CActiveDataProvider.php(178): CDataProvider->getTotalItemCount()
#6 Q:\home\yakutsk-back.dv\framework\web\CDataProvider.php(168): CActiveDataProvider->fetchData()
#7 Q:\home\yakutsk-back.dv\framework\zii\widgets\CBaseListView.php(111): CDataProvider->getData()
#8 Q:\home\yakutsk-back.dv\framework\zii\widgets\CListView.php(211): CBaseListView->init()
#9 Q:\home\yakutsk-back.dv\framework\web\CBaseController.php(147): CListView->init()
#10 Q:\home\yakutsk-back.dv\framework\web\CBaseController.php(180): CBaseController->createWidget('zii.widgets.CLi...', Array)
#11 Q:\home\yakutsk-back.dv\public_html\protected\modules\backend\views\restaurants\index.php(20): CBaseController->widget('zii.widgets.CLi...', Array)
#12 Q:\home\yakutsk-back.dv\framework\web\CBaseController.php(126): require('Q:\home\yakutsk...')
#13 Q:\home\yakutsk-back.dv\framework\web\CBaseController.php(95): CBaseController->renderInternal('Q:\home\yakutsk...', Array, true)
#14 Q:\home\yakutsk-back.dv\framework\web\CController.php(869): CBaseController->renderFile('Q:\home\yakutsk...', Array, true)
#15 Q:\home\yakutsk-back.dv\framework\web\CController.php(782): CController->renderPartial('index', Array, true)
#16 Q:\home\yakutsk-back.dv\public_html\protected\modules\backend\controllers\RestaurantsController.php(128): CController->render('index', Array)
#17 Q:\home\yakutsk-back.dv\framework\web\actions\CInlineAction.php(49): RestaurantsController->actionIndex()
#18 Q:\home\yakutsk-back.dv\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#19 Q:\home\yakutsk-back.dv\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#20 Q:\home\yakutsk-back.dv\framework\web\filters\CFilter.php(40): CFilterChain->run()
#21 Q:\home\yakutsk-back.dv\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#22 Q:\home\yakutsk-back.dv\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#23 Q:\home\yakutsk-back.dv\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#24 Q:\home\yakutsk-back.dv\framework\web\CController.php(291): CFilterChain->run()
#25 Q:\home\yakutsk-back.dv\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#26 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(282): CController->run('')
#27 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('backend/restaur...')
#28 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#29 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#30 {main}
REQUEST_URI=/index.php/backend/restaurants
HTTP_REFERER=http://yakutsk-back.dv/index.php/backend/events
---
2013/09/12 01:35:38 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 01:35:41 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 01:49:20 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 01:51:25 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 01:51:25 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Невозможно обработать запрос "backend/banners".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('backend/banners')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/index.php/backend/banners
HTTP_REFERER=http://yakutsk-back.dv/index.php/backend/company
---
2013/09/12 01:51:26 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 01:51:30 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 01:53:06 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 01:53:21 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 01:53:23 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 01:53:26 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 01:53:30 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 01:53:31 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 01:53:32 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 02:07:45 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 02:07:50 [error] [exception.CException] exception 'CException' with message 'Не определено свойство "Theatre.c_id".' in Q:\home\yakutsk-back.dv\framework\base\CComponent.php:130
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\db\ar\CActiveRecord.php(145): CComponent->__get('c_id')
#1 Q:\home\yakutsk-back.dv\public_html\protected\modules\backend\views\theatre\view.php(7): CActiveRecord->__get('c_id')
#2 Q:\home\yakutsk-back.dv\framework\web\CBaseController.php(126): require('Q:\home\yakutsk...')
#3 Q:\home\yakutsk-back.dv\framework\web\CBaseController.php(95): CBaseController->renderInternal('Q:\home\yakutsk...', Array, true)
#4 Q:\home\yakutsk-back.dv\framework\web\CController.php(869): CBaseController->renderFile('Q:\home\yakutsk...', Array, true)
#5 Q:\home\yakutsk-back.dv\framework\web\CController.php(782): CController->renderPartial('view', Array, true)
#6 Q:\home\yakutsk-back.dv\public_html\protected\modules\backend\controllers\TheatreController.php(56): CController->render('view', Array)
#7 [internal function]: TheatreController->actionView('1')
#8 Q:\home\yakutsk-back.dv\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(TheatreController), Array)
#9 Q:\home\yakutsk-back.dv\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(TheatreController), Object(ReflectionMethod), Array)
#10 Q:\home\yakutsk-back.dv\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#11 Q:\home\yakutsk-back.dv\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#12 Q:\home\yakutsk-back.dv\framework\web\filters\CFilter.php(40): CFilterChain->run()
#13 Q:\home\yakutsk-back.dv\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#14 Q:\home\yakutsk-back.dv\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#15 Q:\home\yakutsk-back.dv\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#16 Q:\home\yakutsk-back.dv\framework\web\CController.php(291): CFilterChain->run()
#17 Q:\home\yakutsk-back.dv\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#18 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(282): CController->run('view')
#19 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('backend/theatre...')
#20 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#21 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#22 {main}
REQUEST_URI=/index.php/backend/theatre/view/id/1
HTTP_REFERER=http://yakutsk-back.dv/index.php/backend/theatre
---
2013/09/12 02:07:50 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 02:07:53 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 02:07:55 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 02:07:59 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 02:08:01 [error] [exception.CException] exception 'CException' with message 'Не определено свойство "Theatre.c_id".' in Q:\home\yakutsk-back.dv\framework\base\CComponent.php:130
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\db\ar\CActiveRecord.php(145): CComponent->__get('c_id')
#1 Q:\home\yakutsk-back.dv\public_html\protected\modules\backend\views\theatre\update.php(7): CActiveRecord->__get('c_id')
#2 Q:\home\yakutsk-back.dv\framework\web\CBaseController.php(126): require('Q:\home\yakutsk...')
#3 Q:\home\yakutsk-back.dv\framework\web\CBaseController.php(95): CBaseController->renderInternal('Q:\home\yakutsk...', Array, true)
#4 Q:\home\yakutsk-back.dv\framework\web\CController.php(869): CBaseController->renderFile('Q:\home\yakutsk...', Array, true)
#5 Q:\home\yakutsk-back.dv\framework\web\CController.php(782): CController->renderPartial('update', Array, true)
#6 Q:\home\yakutsk-back.dv\public_html\protected\modules\backend\controllers\TheatreController.php(103): CController->render('update', Array)
#7 [internal function]: TheatreController->actionUpdate('1')
#8 Q:\home\yakutsk-back.dv\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(TheatreController), Array)
#9 Q:\home\yakutsk-back.dv\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(TheatreController), Object(ReflectionMethod), Array)
#10 Q:\home\yakutsk-back.dv\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#11 Q:\home\yakutsk-back.dv\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#12 Q:\home\yakutsk-back.dv\framework\web\filters\CFilter.php(40): CFilterChain->run()
#13 Q:\home\yakutsk-back.dv\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#14 Q:\home\yakutsk-back.dv\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#15 Q:\home\yakutsk-back.dv\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#16 Q:\home\yakutsk-back.dv\framework\web\CController.php(291): CFilterChain->run()
#17 Q:\home\yakutsk-back.dv\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#18 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(282): CController->run('update')
#19 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('backend/theatre...')
#20 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#21 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#22 {main}
REQUEST_URI=/index.php/backend/theatre/update/id/1
HTTP_REFERER=http://yakutsk-back.dv/index.php/backend/theatre/admin
---
2013/09/12 02:08:01 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 02:08:03 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 02:08:04 [error] [exception.CException] exception 'CException' with message 'Не определено свойство "Theatre.c_id".' in Q:\home\yakutsk-back.dv\framework\base\CComponent.php:130
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\db\ar\CActiveRecord.php(145): CComponent->__get('c_id')
#1 Q:\home\yakutsk-back.dv\public_html\protected\modules\backend\views\theatre\view.php(7): CActiveRecord->__get('c_id')
#2 Q:\home\yakutsk-back.dv\framework\web\CBaseController.php(126): require('Q:\home\yakutsk...')
#3 Q:\home\yakutsk-back.dv\framework\web\CBaseController.php(95): CBaseController->renderInternal('Q:\home\yakutsk...', Array, true)
#4 Q:\home\yakutsk-back.dv\framework\web\CController.php(869): CBaseController->renderFile('Q:\home\yakutsk...', Array, true)
#5 Q:\home\yakutsk-back.dv\framework\web\CController.php(782): CController->renderPartial('view', Array, true)
#6 Q:\home\yakutsk-back.dv\public_html\protected\modules\backend\controllers\TheatreController.php(56): CController->render('view', Array)
#7 [internal function]: TheatreController->actionView('1')
#8 Q:\home\yakutsk-back.dv\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(TheatreController), Array)
#9 Q:\home\yakutsk-back.dv\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(TheatreController), Object(ReflectionMethod), Array)
#10 Q:\home\yakutsk-back.dv\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#11 Q:\home\yakutsk-back.dv\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#12 Q:\home\yakutsk-back.dv\framework\web\filters\CFilter.php(40): CFilterChain->run()
#13 Q:\home\yakutsk-back.dv\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#14 Q:\home\yakutsk-back.dv\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#15 Q:\home\yakutsk-back.dv\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#16 Q:\home\yakutsk-back.dv\framework\web\CController.php(291): CFilterChain->run()
#17 Q:\home\yakutsk-back.dv\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#18 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(282): CController->run('view')
#19 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('backend/theatre...')
#20 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#21 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#22 {main}
REQUEST_URI=/index.php/backend/theatre/view/id/1
HTTP_REFERER=http://yakutsk-back.dv/index.php/backend/theatre/admin
---
2013/09/12 02:08:04 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 02:08:05 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 02:11:15 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 02:11:17 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 02:11:20 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 02:11:23 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 02:11:24 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 02:11:26 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 02:11:28 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 02:11:29 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 02:11:31 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 02:11:34 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 02:11:35 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 02:11:37 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 02:11:40 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 02:11:47 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 02:11:52 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 02:11:57 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 02:12:00 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 02:12:02 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 02:12:04 [error] [exception.CException] exception 'CException' with message 'Не определено свойство "TheatreBanner.c_id".' in Q:\home\yakutsk-back.dv\framework\base\CComponent.php:130
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\db\ar\CActiveRecord.php(145): CComponent->__get('c_id')
#1 Q:\home\yakutsk-back.dv\framework\web\helpers\CHtml.php(2539): CActiveRecord->__get('c_id')
#2 Q:\home\yakutsk-back.dv\framework\web\helpers\CHtml.php(1812): CHtml::resolveValue(Object(TheatreBanner), 'c_id')
#3 Q:\home\yakutsk-back.dv\framework\web\widgets\CActiveForm.php(847): CHtml::activeDropDownList(Object(TheatreBanner), 'c_id', Array, Array)
#4 Q:\home\yakutsk-back.dv\public_html\protected\modules\backend\views\theatreBanner\_form.php(20): CActiveForm->dropDownList(Object(TheatreBanner), 'c_id', Array)
#5 Q:\home\yakutsk-back.dv\framework\web\CBaseController.php(126): require('Q:\home\yakutsk...')
#6 Q:\home\yakutsk-back.dv\framework\web\CBaseController.php(95): CBaseController->renderInternal('Q:\home\yakutsk...', Array, true)
#7 Q:\home\yakutsk-back.dv\framework\web\CController.php(869): CBaseController->renderFile('Q:\home\yakutsk...', Array, true)
#8 Q:\home\yakutsk-back.dv\public_html\protected\modules\backend\views\theatreBanner\create.php(18): CController->renderPartial('_form', Array)
#9 Q:\home\yakutsk-back.dv\framework\web\CBaseController.php(126): require('Q:\home\yakutsk...')
#10 Q:\home\yakutsk-back.dv\framework\web\CBaseController.php(95): CBaseController->renderInternal('Q:\home\yakutsk...', Array, true)
#11 Q:\home\yakutsk-back.dv\framework\web\CController.php(869): CBaseController->renderFile('Q:\home\yakutsk...', Array, true)
#12 Q:\home\yakutsk-back.dv\framework\web\CController.php(782): CController->renderPartial('create', Array, true)
#13 Q:\home\yakutsk-back.dv\public_html\protected\modules\backend\controllers\TheatreBannerController.php(79): CController->render('create', Array)
#14 Q:\home\yakutsk-back.dv\framework\web\actions\CInlineAction.php(49): TheatreBannerController->actionCreate()
#15 Q:\home\yakutsk-back.dv\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#16 Q:\home\yakutsk-back.dv\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#17 Q:\home\yakutsk-back.dv\framework\web\filters\CFilter.php(40): CFilterChain->run()
#18 Q:\home\yakutsk-back.dv\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#19 Q:\home\yakutsk-back.dv\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#20 Q:\home\yakutsk-back.dv\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#21 Q:\home\yakutsk-back.dv\framework\web\CController.php(291): CFilterChain->run()
#22 Q:\home\yakutsk-back.dv\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#23 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(282): CController->run('create')
#24 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('backend/theatre...')
#25 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#26 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#27 {main}
REQUEST_URI=/index.php/backend/theatrebanner/create/id_t/2
HTTP_REFERER=http://yakutsk-back.dv/index.php/backend/theatre/view/id/2
---
2013/09/12 02:12:04 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 02:16:48 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 02:16:52 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 02:16:57 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 02:16:59 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 02:17:00 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 02:17:02 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 02:17:09 [error] [exception.CException] exception 'CException' with message 'Не определено свойство "TheatreBanner.t_published".' in Q:\home\yakutsk-back.dv\framework\base\CComponent.php:130
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\db\ar\CActiveRecord.php(145): CComponent->__get('t_published')
#1 Q:\home\yakutsk-back.dv\framework\validators\CRangeValidator.php(50): CActiveRecord->__get('t_published')
#2 Q:\home\yakutsk-back.dv\framework\validators\CValidator.php(213): CRangeValidator->validateAttribute(Object(TheatreBanner), 't_published')
#3 Q:\home\yakutsk-back.dv\framework\base\CModel.php(159): CValidator->validate(Object(TheatreBanner), NULL)
#4 Q:\home\yakutsk-back.dv\framework\db\ar\CActiveRecord.php(808): CModel->validate(NULL)
#5 Q:\home\yakutsk-back.dv\public_html\protected\modules\backend\controllers\TheatreBannerController.php(73): CActiveRecord->save()
#6 Q:\home\yakutsk-back.dv\framework\web\actions\CInlineAction.php(49): TheatreBannerController->actionCreate()
#7 Q:\home\yakutsk-back.dv\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#8 Q:\home\yakutsk-back.dv\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#9 Q:\home\yakutsk-back.dv\framework\web\filters\CFilter.php(40): CFilterChain->run()
#10 Q:\home\yakutsk-back.dv\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#11 Q:\home\yakutsk-back.dv\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#12 Q:\home\yakutsk-back.dv\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#13 Q:\home\yakutsk-back.dv\framework\web\CController.php(291): CFilterChain->run()
#14 Q:\home\yakutsk-back.dv\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#15 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(282): CController->run('create')
#16 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('backend/theatre...')
#17 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#18 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#19 {main}
REQUEST_URI=/index.php/backend/theatrebanner/create
HTTP_REFERER=http://yakutsk-back.dv/index.php/backend/theatrebanner/create
---
2013/09/12 02:17:09 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 02:18:26 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 02:18:35 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 02:18:41 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 02:18:45 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 02:18:47 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 02:18:52 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 02:18:57 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 02:24:56 [error] [system.db.CDbCommand] Не удалось выполнить CDbCommand::fetchAll(): SQLSTATE[42S02]: Base table or view not found: 1146 Table 'now-yakutsk.restaurants' doesn't exist. Выполнявшийся SQL-запрос: SHOW FULL COLUMNS FROM `restaurants`.
2013/09/12 02:24:56 [error] [exception.CDbException] exception 'CDbException' with message 'Таблица "restaurants", упомянутая в записи active record класса "Restaurants", не найдена в базе данных.' in Q:\home\yakutsk-back.dv\framework\db\ar\CActiveRecord.php:2365
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\db\ar\CActiveRecord.php(411): CActiveRecordMetaData->__construct(Object(Restaurants))
#1 Q:\home\yakutsk-back.dv\framework\db\ar\CActiveRecord.php(661): CActiveRecord->getMetaData()
#2 Q:\home\yakutsk-back.dv\framework\db\ar\CActiveRecord.php(1604): CActiveRecord->getTableSchema()
#3 Q:\home\yakutsk-back.dv\framework\web\CActiveDataProvider.php(228): CActiveRecord->count(Object(CDbCriteria))
#4 Q:\home\yakutsk-back.dv\framework\web\CDataProvider.php(224): CActiveDataProvider->calculateTotalItemCount()
#5 Q:\home\yakutsk-back.dv\framework\web\CActiveDataProvider.php(178): CDataProvider->getTotalItemCount()
#6 Q:\home\yakutsk-back.dv\framework\web\CDataProvider.php(168): CActiveDataProvider->fetchData()
#7 Q:\home\yakutsk-back.dv\framework\zii\widgets\CBaseListView.php(111): CDataProvider->getData()
#8 Q:\home\yakutsk-back.dv\framework\zii\widgets\CListView.php(211): CBaseListView->init()
#9 Q:\home\yakutsk-back.dv\framework\web\CBaseController.php(147): CListView->init()
#10 Q:\home\yakutsk-back.dv\framework\web\CBaseController.php(180): CBaseController->createWidget('zii.widgets.CLi...', Array)
#11 Q:\home\yakutsk-back.dv\public_html\protected\modules\backend\views\restaurants\index.php(20): CBaseController->widget('zii.widgets.CLi...', Array)
#12 Q:\home\yakutsk-back.dv\framework\web\CBaseController.php(126): require('Q:\home\yakutsk...')
#13 Q:\home\yakutsk-back.dv\framework\web\CBaseController.php(95): CBaseController->renderInternal('Q:\home\yakutsk...', Array, true)
#14 Q:\home\yakutsk-back.dv\framework\web\CController.php(869): CBaseController->renderFile('Q:\home\yakutsk...', Array, true)
#15 Q:\home\yakutsk-back.dv\framework\web\CController.php(782): CController->renderPartial('index', Array, true)
#16 Q:\home\yakutsk-back.dv\public_html\protected\modules\backend\controllers\RestaurantsController.php(128): CController->render('index', Array)
#17 Q:\home\yakutsk-back.dv\framework\web\actions\CInlineAction.php(49): RestaurantsController->actionIndex()
#18 Q:\home\yakutsk-back.dv\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#19 Q:\home\yakutsk-back.dv\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#20 Q:\home\yakutsk-back.dv\framework\web\filters\CFilter.php(40): CFilterChain->run()
#21 Q:\home\yakutsk-back.dv\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#22 Q:\home\yakutsk-back.dv\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#23 Q:\home\yakutsk-back.dv\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#24 Q:\home\yakutsk-back.dv\framework\web\CController.php(291): CFilterChain->run()
#25 Q:\home\yakutsk-back.dv\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#26 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(282): CController->run('')
#27 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('backend/restaur...')
#28 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#29 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#30 {main}
REQUEST_URI=/index.php/backend/restaurants
HTTP_REFERER=http://yakutsk-back.dv/index.php/backend/theatrebanner/admin
---
2013/09/12 02:24:56 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 Q:\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('favicon.ico')
#1 Q:\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 Q:\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/favicon.ico
---
2013/09/12 11:13:45 [error] [exception.CException] exception 'CException' with message 'Не определено свойство "Restaurants.t_id".' in D:\Winginx\home\yakutsk-back.dv\framework\base\CComponent.php:130
Stack trace:
#0 D:\Winginx\home\yakutsk-back.dv\framework\db\ar\CActiveRecord.php(145): CComponent->__get('t_id')
#1 D:\Winginx\home\yakutsk-back.dv\public_html\protected\modules\backend\views\restaurants\view.php(7): CActiveRecord->__get('t_id')
#2 D:\Winginx\home\yakutsk-back.dv\framework\web\CBaseController.php(126): require('D:\Winginx\home...')
#3 D:\Winginx\home\yakutsk-back.dv\framework\web\CBaseController.php(95): CBaseController->renderInternal('D:\Winginx\home...', Array, true)
#4 D:\Winginx\home\yakutsk-back.dv\framework\web\CController.php(869): CBaseController->renderFile('D:\Winginx\home...', Array, true)
#5 D:\Winginx\home\yakutsk-back.dv\framework\web\CController.php(782): CController->renderPartial('view', Array, true)
#6 D:\Winginx\home\yakutsk-back.dv\public_html\protected\modules\backend\controllers\RestaurantsController.php(56): CController->render('view', Array)
#7 [internal function]: RestaurantsController->actionView('1')
#8 D:\Winginx\home\yakutsk-back.dv\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(RestaurantsController), Array)
#9 D:\Winginx\home\yakutsk-back.dv\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(RestaurantsController), Object(ReflectionMethod), Array)
#10 D:\Winginx\home\yakutsk-back.dv\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#11 D:\Winginx\home\yakutsk-back.dv\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#12 D:\Winginx\home\yakutsk-back.dv\framework\web\filters\CFilter.php(40): CFilterChain->run()
#13 D:\Winginx\home\yakutsk-back.dv\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#14 D:\Winginx\home\yakutsk-back.dv\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#15 D:\Winginx\home\yakutsk-back.dv\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#16 D:\Winginx\home\yakutsk-back.dv\framework\web\CController.php(291): CFilterChain->run()
#17 D:\Winginx\home\yakutsk-back.dv\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#18 D:\Winginx\home\yakutsk-back.dv\framework\web\CWebApplication.php(282): CController->run('view')
#19 D:\Winginx\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('backend/restaur...')
#20 D:\Winginx\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#21 D:\Winginx\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#22 {main}
REQUEST_URI=/index.php/backend/restaurants/view/id/1
HTTP_REFERER=http://yakutsk-back.dv/index.php/backend/restaurants
---
2013/09/12 11:28:17 [error] [php] Cannot modify header information - headers already sent by (output started at D:\Winginx\home\yakutsk-back.dv\public_html\protected\modules\backend\controllers\RestaurantsController.php:73) (D:\Winginx\home\yakutsk-back.dv\framework\web\CHttpRequest.php:803)
Stack trace:
#0 D:\Winginx\home\yakutsk-back.dv\public_html\protected\modules\backend\controllers\RestaurantsController.php(75): RestaurantsController->redirect()
#1 D:\Winginx\home\yakutsk-back.dv\framework\web\actions\CInlineAction.php(49): RestaurantsController->actionCreate()
#2 D:\Winginx\home\yakutsk-back.dv\framework\web\CController.php(308): CInlineAction->runWithParams()
#3 D:\Winginx\home\yakutsk-back.dv\framework\web\filters\CFilterChain.php(133): RestaurantsController->runAction()
#4 D:\Winginx\home\yakutsk-back.dv\framework\web\filters\CFilter.php(40): CFilterChain->run()
#5 D:\Winginx\home\yakutsk-back.dv\framework\web\CController.php(1145): CAccessControlFilter->filter()
#6 D:\Winginx\home\yakutsk-back.dv\framework\web\filters\CInlineFilter.php(58): RestaurantsController->filterAccessControl()
#7 D:\Winginx\home\yakutsk-back.dv\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#8 D:\Winginx\home\yakutsk-back.dv\framework\web\CController.php(291): CFilterChain->run()
#9 D:\Winginx\home\yakutsk-back.dv\framework\web\CController.php(265): RestaurantsController->runActionWithFilters()
#10 D:\Winginx\home\yakutsk-back.dv\framework\web\CWebApplication.php(282): RestaurantsController->run()
#11 D:\Winginx\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController()
#12 D:\Winginx\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#13 D:\Winginx\home\yakutsk-back.dv\public_html\index.php(10): CWebApplication->run()
REQUEST_URI=/index.php/backend/restaurants/create
2013/09/12 11:28:55 [error] [exception.CException] exception 'CException' with message 'Не определено свойство "RestaurantsBanner.t_id".' in D:\Winginx\home\yakutsk-back.dv\framework\base\CComponent.php:130
Stack trace:
#0 D:\Winginx\home\yakutsk-back.dv\framework\db\ar\CActiveRecord.php(145): CComponent->__get('t_id')
#1 D:\Winginx\home\yakutsk-back.dv\framework\web\helpers\CHtml.php(2539): CActiveRecord->__get('t_id')
#2 D:\Winginx\home\yakutsk-back.dv\framework\web\helpers\CHtml.php(1812): CHtml::resolveValue(Object(RestaurantsBanner), 't_id')
#3 D:\Winginx\home\yakutsk-back.dv\framework\web\widgets\CActiveForm.php(847): CHtml::activeDropDownList(Object(RestaurantsBanner), 't_id', Array, Array)
#4 D:\Winginx\home\yakutsk-back.dv\public_html\protected\modules\backend\views\restaurantsBanner\_form.php(20): CActiveForm->dropDownList(Object(RestaurantsBanner), 't_id', Array)
#5 D:\Winginx\home\yakutsk-back.dv\framework\web\CBaseController.php(126): require('D:\Winginx\home...')
#6 D:\Winginx\home\yakutsk-back.dv\framework\web\CBaseController.php(95): CBaseController->renderInternal('D:\Winginx\home...', Array, true)
#7 D:\Winginx\home\yakutsk-back.dv\framework\web\CController.php(869): CBaseController->renderFile('D:\Winginx\home...', Array, true)
#8 D:\Winginx\home\yakutsk-back.dv\public_html\protected\modules\backend\views\restaurantsBanner\create.php(18): CController->renderPartial('_form', Array)
#9 D:\Winginx\home\yakutsk-back.dv\framework\web\CBaseController.php(126): require('D:\Winginx\home...')
#10 D:\Winginx\home\yakutsk-back.dv\framework\web\CBaseController.php(95): CBaseController->renderInternal('D:\Winginx\home...', Array, true)
#11 D:\Winginx\home\yakutsk-back.dv\framework\web\CController.php(869): CBaseController->renderFile('D:\Winginx\home...', Array, true)
#12 D:\Winginx\home\yakutsk-back.dv\framework\web\CController.php(782): CController->renderPartial('create', Array, true)
#13 D:\Winginx\home\yakutsk-back.dv\public_html\protected\modules\backend\controllers\RestaurantsBannerController.php(79): CController->render('create', Array)
#14 D:\Winginx\home\yakutsk-back.dv\framework\web\actions\CInlineAction.php(49): RestaurantsBannerController->actionCreate()
#15 D:\Winginx\home\yakutsk-back.dv\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#16 D:\Winginx\home\yakutsk-back.dv\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#17 D:\Winginx\home\yakutsk-back.dv\framework\web\filters\CFilter.php(40): CFilterChain->run()
#18 D:\Winginx\home\yakutsk-back.dv\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#19 D:\Winginx\home\yakutsk-back.dv\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#20 D:\Winginx\home\yakutsk-back.dv\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#21 D:\Winginx\home\yakutsk-back.dv\framework\web\CController.php(291): CFilterChain->run()
#22 D:\Winginx\home\yakutsk-back.dv\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#23 D:\Winginx\home\yakutsk-back.dv\framework\web\CWebApplication.php(282): CController->run('create')
#24 D:\Winginx\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('backend/restaur...')
#25 D:\Winginx\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#26 D:\Winginx\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#27 {main}
REQUEST_URI=/index.php/backend/restaurantsbanner/create/id_r/2
HTTP_REFERER=http://yakutsk-back.dv/index.php/backend/restaurants/view/id/2
---
2013/09/12 11:36:07 [error] [exception.CException] exception 'CException' with message 'Не определено свойство "Theatre.r_name".' in D:\Winginx\home\yakutsk-back.dv\framework\base\CComponent.php:130
Stack trace:
#0 D:\Winginx\home\yakutsk-back.dv\framework\db\ar\CActiveRecord.php(145): CComponent->__get('r_name')
#1 D:\Winginx\home\yakutsk-back.dv\public_html\protected\modules\backend\views\restaurantsBanner\_view.php(15): CActiveRecord->__get('r_name')
#2 D:\Winginx\home\yakutsk-back.dv\framework\web\CBaseController.php(130): require('D:\Winginx\home...')
#3 D:\Winginx\home\yakutsk-back.dv\framework\web\CBaseController.php(95): CBaseController->renderInternal('D:\Winginx\home...', Array, false)
#4 D:\Winginx\home\yakutsk-back.dv\framework\zii\widgets\CListView.php(291): CBaseController->renderFile('D:\Winginx\home...', Array)
#5 D:\Winginx\home\yakutsk-back.dv\framework\zii\widgets\CBaseListView.php(167): CListView->renderItems()
#6 [internal function]: CBaseListView->renderSection(Array)
#7 D:\Winginx\home\yakutsk-back.dv\framework\zii\widgets\CBaseListView.php(150): preg_replace_callback('/{(\w+)}/', Array, '{summary}?{sort...')
#8 D:\Winginx\home\yakutsk-back.dv\framework\zii\widgets\CBaseListView.php(135): CBaseListView->renderContent()
#9 D:\Winginx\home\yakutsk-back.dv\framework\web\CBaseController.php(181): CBaseListView->run()
#10 D:\Winginx\home\yakutsk-back.dv\public_html\protected\modules\backend\views\restaurantsBanner\index.php(20): CBaseController->widget('zii.widgets.CLi...', Array)
#11 D:\Winginx\home\yakutsk-back.dv\framework\web\CBaseController.php(126): require('D:\Winginx\home...')
#12 D:\Winginx\home\yakutsk-back.dv\framework\web\CBaseController.php(95): CBaseController->renderInternal('D:\Winginx\home...', Array, true)
#13 D:\Winginx\home\yakutsk-back.dv\framework\web\CController.php(869): CBaseController->renderFile('D:\Winginx\home...', Array, true)
#14 D:\Winginx\home\yakutsk-back.dv\framework\web\CController.php(782): CController->renderPartial('index', Array, true)
#15 D:\Winginx\home\yakutsk-back.dv\public_html\protected\modules\backend\controllers\RestaurantsBannerController.php(128): CController->render('index', Array)
#16 D:\Winginx\home\yakutsk-back.dv\framework\web\actions\CInlineAction.php(49): RestaurantsBannerController->actionIndex()
#17 D:\Winginx\home\yakutsk-back.dv\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#18 D:\Winginx\home\yakutsk-back.dv\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#19 D:\Winginx\home\yakutsk-back.dv\framework\web\filters\CFilter.php(40): CFilterChain->run()
#20 D:\Winginx\home\yakutsk-back.dv\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#21 D:\Winginx\home\yakutsk-back.dv\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#22 D:\Winginx\home\yakutsk-back.dv\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#23 D:\Winginx\home\yakutsk-back.dv\framework\web\CController.php(291): CFilterChain->run()
#24 D:\Winginx\home\yakutsk-back.dv\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#25 D:\Winginx\home\yakutsk-back.dv\framework\web\CWebApplication.php(282): CController->run('index')
#26 D:\Winginx\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('backend/restaur...')
#27 D:\Winginx\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#28 D:\Winginx\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#29 {main}
REQUEST_URI=/index.php/backend/restaurantsbanner/index
HTTP_REFERER=http://yakutsk-back.dv/index.php/backend/restaurantsbanner/create
---
2013/09/12 11:46:53 [error] [exception.CException] exception 'CException' with message 'Не определено свойство "Events.c_published".' in D:\Winginx\home\yakutsk-back.dv\framework\base\CComponent.php:130
Stack trace:
#0 D:\Winginx\home\yakutsk-back.dv\framework\db\ar\CActiveRecord.php(145): CComponent->__get('c_published')
#1 D:\Winginx\home\yakutsk-back.dv\public_html\protected\modules\backend\views\events\_form.php(77): CActiveRecord->__get('c_published')
#2 D:\Winginx\home\yakutsk-back.dv\framework\web\CBaseController.php(126): require('D:\Winginx\home...')
#3 D:\Winginx\home\yakutsk-back.dv\framework\web\CBaseController.php(95): CBaseController->renderInternal('D:\Winginx\home...', Array, true)
#4 D:\Winginx\home\yakutsk-back.dv\framework\web\CController.php(869): CBaseController->renderFile('D:\Winginx\home...', Array, true)
#5 D:\Winginx\home\yakutsk-back.dv\public_html\protected\modules\backend\views\events\create.php(18): CController->renderPartial('_form', Array)
#6 D:\Winginx\home\yakutsk-back.dv\framework\web\CBaseController.php(126): require('D:\Winginx\home...')
#7 D:\Winginx\home\yakutsk-back.dv\framework\web\CBaseController.php(95): CBaseController->renderInternal('D:\Winginx\home...', Array, true)
#8 D:\Winginx\home\yakutsk-back.dv\framework\web\CController.php(869): CBaseController->renderFile('D:\Winginx\home...', Array, true)
#9 D:\Winginx\home\yakutsk-back.dv\framework\web\CController.php(782): CController->renderPartial('create', Array, true)
#10 D:\Winginx\home\yakutsk-back.dv\public_html\protected\modules\backend\controllers\EventsController.php(80): CController->render('create', Array)
#11 D:\Winginx\home\yakutsk-back.dv\framework\web\actions\CInlineAction.php(49): EventsController->actionCreate()
#12 D:\Winginx\home\yakutsk-back.dv\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#13 D:\Winginx\home\yakutsk-back.dv\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#14 D:\Winginx\home\yakutsk-back.dv\framework\web\filters\CFilter.php(40): CFilterChain->run()
#15 D:\Winginx\home\yakutsk-back.dv\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#16 D:\Winginx\home\yakutsk-back.dv\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#17 D:\Winginx\home\yakutsk-back.dv\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#18 D:\Winginx\home\yakutsk-back.dv\framework\web\CController.php(291): CFilterChain->run()
#19 D:\Winginx\home\yakutsk-back.dv\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#20 D:\Winginx\home\yakutsk-back.dv\framework\web\CWebApplication.php(282): CController->run('create')
#21 D:\Winginx\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('backend/events/...')
#22 D:\Winginx\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#23 D:\Winginx\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#24 {main}
REQUEST_URI=/index.php/backend/events/create
HTTP_REFERER=http://yakutsk-back.dv/index.php/backend/events
---
2013/09/12 11:47:27 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested page does not exist.' in D:\Winginx\home\yakutsk-back.dv\public_html\protected\modules\backend\controllers\EventsController.php:174
Stack trace:
#0 D:\Winginx\home\yakutsk-back.dv\public_html\protected\modules\backend\controllers\EventsController.php(90): EventsController->loadModel('img.jpg')
#1 [internal function]: EventsController->actionUpdate('img.jpg')
#2 D:\Winginx\home\yakutsk-back.dv\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(EventsController), Array)
#3 D:\Winginx\home\yakutsk-back.dv\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(EventsController), Object(ReflectionMethod), Array)
#4 D:\Winginx\home\yakutsk-back.dv\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 D:\Winginx\home\yakutsk-back.dv\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 D:\Winginx\home\yakutsk-back.dv\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 D:\Winginx\home\yakutsk-back.dv\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 D:\Winginx\home\yakutsk-back.dv\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 D:\Winginx\home\yakutsk-back.dv\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 D:\Winginx\home\yakutsk-back.dv\framework\web\CController.php(291): CFilterChain->run()
#11 D:\Winginx\home\yakutsk-back.dv\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 D:\Winginx\home\yakutsk-back.dv\framework\web\CWebApplication.php(282): CController->run('update')
#13 D:\Winginx\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('backend/events/...')
#14 D:\Winginx\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#15 D:\Winginx\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#16 {main}
REQUEST_URI=/index.php/backend/events/update/id/img.jpg
HTTP_REFERER=http://yakutsk-back.dv/index.php/backend/events/update/id/1
---
2013/09/12 11:54:30 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Невозможно обработать запрос "backend/beautyandhealth".' in D:\Winginx\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\Winginx\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('backend/beautya...')
#1 D:\Winginx\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\Winginx\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/index.php/backend/beautyandhealth
HTTP_REFERER=http://yakutsk-back.dv/index.php/backend/events/index
---
2013/09/12 12:04:02 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested page does not exist.' in D:\Winginx\home\yakutsk-back.dv\public_html\protected\modules\backend\controllers\BeautyandhealthController.php:157
Stack trace:
#0 D:\Winginx\home\yakutsk-back.dv\public_html\protected\modules\backend\controllers\BeautyandhealthController.php(89): BeautyandhealthController->loadModel('img.jpg')
#1 [internal function]: BeautyandhealthController->actionUpdate('img.jpg')
#2 D:\Winginx\home\yakutsk-back.dv\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(BeautyandhealthController), Array)
#3 D:\Winginx\home\yakutsk-back.dv\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(BeautyandhealthController), Object(ReflectionMethod), Array)
#4 D:\Winginx\home\yakutsk-back.dv\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 D:\Winginx\home\yakutsk-back.dv\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 D:\Winginx\home\yakutsk-back.dv\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 D:\Winginx\home\yakutsk-back.dv\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 D:\Winginx\home\yakutsk-back.dv\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 D:\Winginx\home\yakutsk-back.dv\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 D:\Winginx\home\yakutsk-back.dv\framework\web\CController.php(291): CFilterChain->run()
#11 D:\Winginx\home\yakutsk-back.dv\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 D:\Winginx\home\yakutsk-back.dv\framework\web\CWebApplication.php(282): CController->run('update')
#13 D:\Winginx\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('backend/beautya...')
#14 D:\Winginx\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#15 D:\Winginx\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#16 {main}
REQUEST_URI=/index.php/backend/beautyandhealth/update/id/img.jpg
HTTP_REFERER=http://yakutsk-back.dv/index.php/backend/beautyandhealth/update/id/1
---
2013/09/12 12:14:43 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Невозможно обработать запрос "backend/banners".' in D:\Winginx\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\Winginx\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('backend/banners')
#1 D:\Winginx\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\Winginx\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/index.php/backend/banners
HTTP_REFERER=http://yakutsk-back.dv/index.php/backend/beautyandhealth/view/id/1
---
2013/09/12 12:15:39 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested page does not exist.' in D:\Winginx\home\yakutsk-back.dv\public_html\protected\modules\backend\controllers\EntertainmentController.php:157
Stack trace:
#0 D:\Winginx\home\yakutsk-back.dv\public_html\protected\modules\backend\controllers\EntertainmentController.php(89): EntertainmentController->loadModel('poster.jpg')
#1 [internal function]: EntertainmentController->actionUpdate('poster.jpg')
#2 D:\Winginx\home\yakutsk-back.dv\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(EntertainmentController), Array)
#3 D:\Winginx\home\yakutsk-back.dv\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(EntertainmentController), Object(ReflectionMethod), Array)
#4 D:\Winginx\home\yakutsk-back.dv\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 D:\Winginx\home\yakutsk-back.dv\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 D:\Winginx\home\yakutsk-back.dv\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 D:\Winginx\home\yakutsk-back.dv\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 D:\Winginx\home\yakutsk-back.dv\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 D:\Winginx\home\yakutsk-back.dv\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 D:\Winginx\home\yakutsk-back.dv\framework\web\CController.php(291): CFilterChain->run()
#11 D:\Winginx\home\yakutsk-back.dv\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 D:\Winginx\home\yakutsk-back.dv\framework\web\CWebApplication.php(282): CController->run('update')
#13 D:\Winginx\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('backend/enterta...')
#14 D:\Winginx\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#15 D:\Winginx\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#16 {main}
REQUEST_URI=/index.php/backend/entertainment/update/id/poster.jpg
HTTP_REFERER=http://yakutsk-back.dv/index.php/backend/entertainment/update/id/1
---
2013/09/12 12:15:54 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "2.png".' in D:\Winginx\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\Winginx\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('2.png')
#1 D:\Winginx\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\Winginx\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/2.png
HTTP_REFERER=http://yakutsk-back.dv/index.php/backend/entertainment/update/id/1
---
2013/09/12 12:16:29 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested page does not exist.' in D:\Winginx\home\yakutsk-back.dv\public_html\protected\modules\backend\controllers\EntertainmentController.php:157
Stack trace:
#0 D:\Winginx\home\yakutsk-back.dv\public_html\protected\modules\backend\controllers\EntertainmentController.php(89): EntertainmentController->loadModel('poster.jpg')
#1 [internal function]: EntertainmentController->actionUpdate('poster.jpg')
#2 D:\Winginx\home\yakutsk-back.dv\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(EntertainmentController), Array)
#3 D:\Winginx\home\yakutsk-back.dv\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(EntertainmentController), Object(ReflectionMethod), Array)
#4 D:\Winginx\home\yakutsk-back.dv\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 D:\Winginx\home\yakutsk-back.dv\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 D:\Winginx\home\yakutsk-back.dv\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 D:\Winginx\home\yakutsk-back.dv\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 D:\Winginx\home\yakutsk-back.dv\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 D:\Winginx\home\yakutsk-back.dv\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 D:\Winginx\home\yakutsk-back.dv\framework\web\CController.php(291): CFilterChain->run()
#11 D:\Winginx\home\yakutsk-back.dv\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 D:\Winginx\home\yakutsk-back.dv\framework\web\CWebApplication.php(282): CController->run('update')
#13 D:\Winginx\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('backend/enterta...')
#14 D:\Winginx\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#15 D:\Winginx\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#16 {main}
REQUEST_URI=/index.php/backend/entertainment/update/id/poster.jpg
HTTP_REFERER=http://yakutsk-back.dv/index.php/backend/entertainment/update/id/1
---
2013/09/12 12:28:32 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Невозможно обработать запрос "backend/img.jpg".' in D:\Winginx\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\Winginx\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('backend/img.jpg')
#1 D:\Winginx\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\Winginx\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/index.php/backend/img.jpg
HTTP_REFERER=http://yakutsk-back.dv/index.php/backend/shipment
---
2013/09/12 12:28:53 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested page does not exist.' in D:\Winginx\home\yakutsk-back.dv\public_html\protected\modules\backend\controllers\ShipmentController.php:157
Stack trace:
#0 D:\Winginx\home\yakutsk-back.dv\public_html\protected\modules\backend\controllers\ShipmentController.php(89): ShipmentController->loadModel('img.jpg')
#1 [internal function]: ShipmentController->actionUpdate('img.jpg')
#2 D:\Winginx\home\yakutsk-back.dv\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(ShipmentController), Array)
#3 D:\Winginx\home\yakutsk-back.dv\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(ShipmentController), Object(ReflectionMethod), Array)
#4 D:\Winginx\home\yakutsk-back.dv\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 D:\Winginx\home\yakutsk-back.dv\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 D:\Winginx\home\yakutsk-back.dv\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 D:\Winginx\home\yakutsk-back.dv\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 D:\Winginx\home\yakutsk-back.dv\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 D:\Winginx\home\yakutsk-back.dv\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 D:\Winginx\home\yakutsk-back.dv\framework\web\CController.php(291): CFilterChain->run()
#11 D:\Winginx\home\yakutsk-back.dv\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 D:\Winginx\home\yakutsk-back.dv\framework\web\CWebApplication.php(282): CController->run('update')
#13 D:\Winginx\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('backend/shipmen...')
#14 D:\Winginx\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#15 D:\Winginx\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#16 {main}
REQUEST_URI=/index.php/backend/shipment/update/id/img.jpg
HTTP_REFERER=http://yakutsk-back.dv/index.php/backend/shipment/update/id/1
---
2013/09/12 14:57:00 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Невозможно обработать запрос "backend/hightlife".' in D:\Winginx\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\Winginx\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('backend/hightli...')
#1 D:\Winginx\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\Winginx\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/index.php/backend/hightlife
HTTP_REFERER=http://yakutsk-back.dv/index.php/backend/cinema
---
2013/09/12 18:04:23 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Невозможно обработать запрос "backend/filmsbanner/create/id_c/1".' in D:\Winginx\home\yakutsk-back.dv\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\Winginx\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('backend/filmsba...')
#1 D:\Winginx\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\Winginx\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#3 {main}
REQUEST_URI=/index.php/backend/filmsbanner/create/id_c/1
HTTP_REFERER=http://yakutsk-back.dv/index.php/backend/cinema/view/id/1
---
2013/09/12 18:14:59 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested page does not exist.' in D:\Winginx\home\yakutsk-back.dv\public_html\protected\modules\backend\controllers\FilmsController.php:157
Stack trace:
#0 D:\Winginx\home\yakutsk-back.dv\public_html\protected\modules\backend\controllers\FilmsController.php(89): FilmsController->loadModel('kino3.png')
#1 [internal function]: FilmsController->actionUpdate('kino3.png')
#2 D:\Winginx\home\yakutsk-back.dv\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(FilmsController), Array)
#3 D:\Winginx\home\yakutsk-back.dv\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(FilmsController), Object(ReflectionMethod), Array)
#4 D:\Winginx\home\yakutsk-back.dv\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 D:\Winginx\home\yakutsk-back.dv\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 D:\Winginx\home\yakutsk-back.dv\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 D:\Winginx\home\yakutsk-back.dv\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 D:\Winginx\home\yakutsk-back.dv\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 D:\Winginx\home\yakutsk-back.dv\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 D:\Winginx\home\yakutsk-back.dv\framework\web\CController.php(291): CFilterChain->run()
#11 D:\Winginx\home\yakutsk-back.dv\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 D:\Winginx\home\yakutsk-back.dv\framework\web\CWebApplication.php(282): CController->run('update')
#13 D:\Winginx\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('backend/films/u...')
#14 D:\Winginx\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#15 D:\Winginx\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#16 {main}
REQUEST_URI=/index.php/backend/films/update/id/kino3.png
HTTP_REFERER=http://yakutsk-back.dv/index.php/backend/films/update/id/3
---
2013/09/12 18:15:14 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested page does not exist.' in D:\Winginx\home\yakutsk-back.dv\public_html\protected\modules\backend\controllers\FilmsController.php:157
Stack trace:
#0 D:\Winginx\home\yakutsk-back.dv\public_html\protected\modules\backend\controllers\FilmsController.php(89): FilmsController->loadModel('kino3.png')
#1 [internal function]: FilmsController->actionUpdate('kino3.png')
#2 D:\Winginx\home\yakutsk-back.dv\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(FilmsController), Array)
#3 D:\Winginx\home\yakutsk-back.dv\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(FilmsController), Object(ReflectionMethod), Array)
#4 D:\Winginx\home\yakutsk-back.dv\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 D:\Winginx\home\yakutsk-back.dv\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 D:\Winginx\home\yakutsk-back.dv\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 D:\Winginx\home\yakutsk-back.dv\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 D:\Winginx\home\yakutsk-back.dv\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 D:\Winginx\home\yakutsk-back.dv\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 D:\Winginx\home\yakutsk-back.dv\framework\web\CController.php(291): CFilterChain->run()
#11 D:\Winginx\home\yakutsk-back.dv\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 D:\Winginx\home\yakutsk-back.dv\framework\web\CWebApplication.php(282): CController->run('update')
#13 D:\Winginx\home\yakutsk-back.dv\framework\web\CWebApplication.php(141): CWebApplication->runController('backend/films/u...')
#14 D:\Winginx\home\yakutsk-back.dv\framework\base\CApplication.php(180): CWebApplication->processRequest()
#15 D:\Winginx\home\yakutsk-back.dv\public_html\index.php(10): CApplication->run()
#16 {main}
REQUEST_URI=/index.php/backend/films/update/id/kino3.png
HTTP_REFERER=http://yakutsk-back.dv/index.php/backend/films/update/id/3
---
